docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -182,14 +194,14 @@ You can modify the flex container appearance and behavior through the following
 
     **Usage:**
 
-    ```
+    ```csharp
     FlexContainer flexContainer = new FlexContainer();
     flexContainer.ContentDirection = FlexContainer.ContentDirectionType.RTL;
     ```
 
 -   `FlexDirection`
 
-    This property specifies the main axis direction along which flex items are placed.
+    This property specifies the main axis direction along which flex items are placed. The default property is `Column`.
```

Comment:
Since the sentence is immediately followed by an image, so we need : instead of .
https://github.com/Samsung/tizen-docs/pull/1272/commits/c850e531bc65eff63e37543f732ae4c2dc1f4744#r505164016

Reply:
So I will apply it in all sentences, which is immediately followed by an image

Reply:
@pkalota Yes, if a sentence(s) is followed image or code then **:** is used